### PR TITLE
Refactor - Scripts File Cleanup

### PR DESCRIPTION
**Pull request by jaypeasee**
# Pull Request Template

#### What’s this PR do?
This is a cleanup of `scripts.js` Specifically this branch: 
1. puts the User Data fetch in a function to be invoked on load.
1. reorders where functions are in relation to others in order to make the file more readable.
1. reorders where variable declarations and callbacks go within each function.
1. cleans up rogue semi colons

#### Where should the reviewer start?
Review all of `scripts.js`

#### How should this be manually tested?
1. Load the page a bunch of times.
1. Submit all of the forms.
Make sure nothing new breaks

### CODE REVIEW

**Comment by jdxsmith** [comment]
- The flow of our functions looks WAY better, nice work.  
- The little things like semi-colons and spacing add some developer empathy as well, so good job on that.

File: src/scripts.js [hunk 1]
```diff
@@ -16,164 +14,61 @@ const userForms = document.querySelector('.todays-metrics');
...
-//window.addEventListener('load', defineVariables);
-//window.addEventListener('load', getApiData);
 //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-
-let userData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData')
-  .then(response => response.json())
-  .then(data => createRandomUser(data.userData))
-  .catch(error => console.log(error.message))
+function fetchUsers() {
```
Comment (Codo-Baggins) [inline src/scripts.js:22]: I think capturing this in a function was smart.

File: src/scripts.js [hunk 2]
```diff
@@ -16,164 +14,61 @@ const userForms = document.querySelector('.todays-metrics');
...
+  return listRepo.getDataFromID(id);
+};
 
+function loadMonitorData() {
   let sleepData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/sleep/sleepData')
     .then(response => response.json())
     .then(data => data.sleepData)
-    // .catch(error => console.log("error.message"))
-
+    .catch(error => console.log("error.message"))
```
Comment (Codo-Baggins) [inline src/scripts.js:57]: I think the "error message" should be removed from quotes.

File: src/scripts.js [hunk 3]
```diff
@@ -16,164 +14,61 @@ const userForms = document.querySelector('.todays-metrics');
...
-  console.log(userNowId)
-  historicalWeek.forEach(instance => instance.insertAdjacentHTML('afterBegin', `Week of ${randomHistory}`));
-  addInfoToSidebar(userNow, userRepo);
-  getHyrdationElements(userNowId, hydrationRepo, today, userRepo, randomHistory)
-  getSleepElements(userNowId, sleepRepo, today, userRepo, randomHistory);
-  let winnerNow = makeWinnerID(activityRepo, userNow, today, userRepo);
-  getFriendInfoElements(userNowId, activityRepo, userRepo, today, randomHistory, userNow);
-  getActivityElements(userNowId, activityRepo, today, userRepo, randomHistory, userNow, winnerNow);
-}
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (Codo-Baggins) [inline src/scripts.js:71]: I really like your use of a physical barrier between different sections of code.

File: src/scripts.js [hunk 4]
```diff
@@ -346,188 +232,302 @@ function makeSleepHTML(id, sleepInfo, userStorage, method) {
...
+  	'Content-Type': 'application/json'
+    },
+      body: JSON.stringify(dataToPost),
+    })
+    .then(response => response.json())
+    .then(message => handlePostSuccess('.activity'))
+    .catch(error => console.log(error.message))
+  }
+
+//not being used!
```
Comment (Codo-Baggins) [inline src/scripts.js:530]: I saw this too.  I think we need to have a conversation about what the purpose of this function was supposed to be before entirely removing it, but nice catch!

**Review by Codo-Baggins — APPROVED** [review]
Great refactoring overall.  Good work reformatting, reorganizing, and adding semi-colons where applicable.

**Pull request merged**